lib/fetching/content-services/crowd-tangle-content-service.js

```diff
@@ -184,7 +185,20 @@ CrowdTangleContentService.prototype._parse = function(data, options) {
 	if (this.crowdtangle_lists[data.account.id]) {
 		metadata.ct_tag = this.crowdtangle_lists[data.account.id];
 	} else if (options.requestType == "SAVED_SEARCH") {
-		metadata.ct_tag = "Saved Search";
+		metadata.ct_tag = []
+		for(var saved_search_id in this.crowdtangle_saved_searches){
```

Comment:
Clever workaround, I'm glad this works! For the sake of others reading this later, could you please add a comment here explaining that this logic searches for a match to determine if it's likely related to a saved search, and also note that it's a workaround that we hope will go away once CT supports it?